doc/tutorials/image/image.txt

```diff
@@ -0,0 +1,210 @@
+/*!
+@file image.txt
+@author Mehul Kumar Nirala
+@brief Tutorial for how to load and save images in mlpack.
+
+@page imagetutorial Image Utilities tutorial
+
+@section intro_imagetut Introduction
+
+Image dataset are becoming increasingly popular in deep learning.
+
+mlpack currently implements saving and loading of image. It uses [stb/](https://github.com/nothings/stb) to achieve this.
+
+@section toc_imagetut Table of Contents
+
+This tutorial is split into the following sections:
+
+ - \ref intro_imagetut
+ - \ref toc_imagetut
+ - \ref model_api_imagetut
+ - \ref imageinfo_api_imagetut
+ - \ref load_api_imagetut
+ - \ref save_api_imagetut
+
+@section model_api_imagetut Model API
+
+Image utilities supports loading and saving of image. Images can also be loaded from a directory if CXX version >= 17 as it requires <filesystem> header to navigate a directory.

```

Comment:
Not sure everyone knows what CXX version is, maybe we should go with: "if the compiler supports c++17 as the <filesystem> functionality is required"?